src/index.html

```diff
@@ -7,12 +7,55 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="product_image"
+      />
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <div
+        class="card__code"
+        data-qa="card"
```

Comment:
The 'data-qa' attribute is usually used for testing purposes and should not be present in the production code unless it is specifically required for tests. Make sure it aligns with the project requirements.